CONTRIBUTING.md

```diff
@@ -1,501 +1,108 @@
+Avoid superficial changes aimed at farming airdrops or karma.
 
-## Style Guides
+## Communication
 
-### Git Commit Messages
+See the [Community section](./README.md#community) in our README for Discord,
+GitHub discussions, and other communication channels.
 
-The gno project tends to use the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard for any
-commit that goes into the main code stream (currently, the `master` branch).
+---
 
-Each PR is squashed and merged into the main code stream, which means PR _titles_ should adhere to the Conventional
-Commit standard, and be short and precise. This is practically enforced using a
-[linter check](https://github.com/amannn/action-semantic-pull-request). The type
-of PRs that are allowed are the following:
-
-* **build**\
-  Changes that affect the build system or external dependencies.
-* **chore**\
-  Other changes that don't modify source or test files.
-* **ci**\
-  Changes to our CI configuration files (ie. GitHub Actions) and scripts.
-* **docs**\
-  Documentation only changes.
-* **feat**\
-  A new feature.
-* **fix**\
-  A bug fix.
-* **perf**\
-  A code change that improves performance.
-* **refactor**\
-  A code change that neither fixes a bug nor adds a feature.
-* **revert**\
-  Reverts a previous commit.
-* **style**\
-  Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc).
-* **test**\
-  Adding missing tests or correcting existing tests.
-
-A general rule of thumb:
-
-- Never favor rewriting history in PRs (rebases have very few exceptions, like implementation rewrites;
-see [A Word on Rebasing](#a-word-on-rebasing))
-- Tend to make separate commits for logically separate changes
-
-### Go/Gno Style Guide
-
-The gno codebase currently doesn’t follow any specific standard in terms of code style. The core team, and core
-contributors, favor using the following code style guides:
-
-- [Effective Go](https://go.dev/doc/effective_go)
-- [Uber’s Go Style Guide](https://github.com/uber-go/guide/blob/master/style.md)
-- [Google’s Go Style Guide](https://google.github.io/styleguide/go/guide)
-  and [Google’s Go Best Practices](https://google.github.io/eng-practices/review/)
-
-If unsure, you should always follow the existing code style of the repository.
-
-Additionally, the gno codebase uses linters to enforce some common coding style etiquette. Any PR that aims to modify
-the gno codebase should make sure that the linter checks pass; otherwise they won’t be merged into the main code stream.
-
-### Documentation Style Guide
-
-When writing in-code documentation, always favor to stay aligned with the godoc standard. Gno’s development philosophy
-stays true to idiomatic Go, and this transfers to the documentation as well.
-
-Resources for idiomatic Go docs:
-
-- [godoc](https://go.dev/blog/godoc)
-- [Go Doc Comments](https://tip.golang.org/doc/comment)
-
-## Avoiding Unhelpful Contributions
```

Comment:
I sometimes refer to this section when closing airdrop farming PR's. But maybe it's best to remove it as the people who do airdrop farming don't care about contributing, anyway.